packages/jest-puppeteer-axe/src/index.js

```diff
@@ -80,7 +80,7 @@ async function toPassAxeTests(
 	page,
 	{ include, exclude, disabledRules, options, config } = {}
 ) {
-	const axe = new AxePuppeteer( page );
+	const axe = await new AxePuppeteer( page );
```

Comment:
I noticed missing `await` when looking at docs:

https://github.com/dequelabs/axe-core-npm/tree/develop/packages/puppeteer#usage

Reply:
Ah, wait, it was because of `analyze`, I will revert :)

```suggestion
	const axe = new AxePuppeteer( page );
```